### ISSUE TITLE
Support for multiple languages

### ISSUE THREAD

**Issue by Lifer233** [harjot-oberai/MusicStreamer#57]
I think this app should add other language.

**Comment by harjot-oberai** [comment]
Well I can't do it myself, if you would like to help, you are most welcome.

Cheers 🍺

**Comment by mehulTank** [comment]
yes i am interested to contribute my self. please let me know which language want to add here

**Comment by harjot-oberai** [comment]
@mehulTank : Whichever language you are fluent in 😄 

It will take time and a lot of effort, Some strings are hardcoded and you'll need to add reference for them in the **strings.xml**.

Thanks for your help though
Cheers! 🍻
